pallets/connectors-gateway/gateway-routers/src/moonbeam.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
```

Comment:
Sounds reasonable.

Reply:
Are there any other parachains that this would work with? Otherwise I would keep this as it is in order to make sure there's a clear distinction between the `ethereum_xcm` setups that can be available.

Reply:
No, we only integrate with Moonbeam at the moment. I am fine with keeping it as is 👍

Reply:
Let's keep it like this for now, maybe we'll find a more fitting name after we add more routers.